### PR TITLE
fix(Pairing): Added installation info to pairing results

### PR DESCRIPTION
**Pull request by igor-sirotin**
Fixes https://github.com/status-im/status-desktop/issues/9930
Requires https://github.com/status-im/status-go/pull/3422

### What does the PR do

- Added synced device installation info to sync devices popup
- Updated local pairing start/end detection logic
- _minor:_ Force active input in device name popup when open

NOTE:
During onboarding, it's not possible to see synced device info, because it's transferred as the last operation of pairing. And immediately after that the app auto-logins. This should be improved with https://github.com/status-im/status-go/issues/3399.

### Affected areas

* settings -> syncing
* onboaring -> sing in with sync code

### Screenshot of functionality (including design for comparison)

Screenshot 2023-04-24 at 23 42 00

### CODE REVIEW

File: src/app/modules/startup/controller.nim [hunk 1]
```diff
@@ -167,6 +167,7 @@ proc init*(self: Controller) =
   self.connectionIds.add(handlerId)
 
   handlerId = self.events.onWithUUID(SIGNAL_LOCAL_PAIRING_STATUS_UPDATE) do(e: Args):
+    echo "<<< on SIGNAL_LOCAL_PAIRING_STATUS_UPDATE"
```
Comment (caybro) [inline src/app/modules/startup/controller.nim:170]: leftover?

**Review by caybro — APPROVED** [review]
LGTM

File: src/app/core/signals/remote_signals/pairing.nim [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
-  eventType*: string
-  action*: int
+  eventType*: EventType
+  action*: Action
   error*: string
   account*: AccountDto
+  installation*: InstallationDto
 
 proc fromEvent*(T: type LocalPairingSignal, event: JsonNode): LocalPairingSignal =
+  echo "<<< pairing event: " & $event
```
Comment (osmaczko) [inline src/app/core/signals/remote_signals/pairing.nim:16]: leftover?

**Review by osmaczko — APPROVED** [review]
:+1:

**Pull request merged**